examples/examples.go

```diff
@@ -46,7 +46,7 @@ func ServiceAccount(flowClient *client.Client) (flow.Address, *flow.AccountKey,
 	privateKey, _ := crypto.GeneratePrivateKey(crypto.ECDSA_P256, []byte(defaultServiceKeySeed))
 
 	// Service address is the first generated account address
```

Comment:
Maybe point out that this needs to be changed if run against mainnet

Reply:
Good idea! I'm going to follow up with more extensive documentation on the new address format and how to switch between chains.